pkg/sensors/tracing/policyfilter_test.go

```diff
@@ -65,7 +65,7 @@ func createCgroup(t *testing.T, dir string, pids ...uint64) policyfilter.CgroupI
 		// NB(kkourt): this is just for our vmtests VM
 		cmd := exec.Command("sudo", "mount", "-o", "remount,rw", cgroupFs)
 		cmd.Run()
-		control, err := cgroups.New(cgroups.Systemd, cgroups.Slice(slice, dir), &specs.LinuxResources{
+		control, err := cgroups.New(cgroups.V1, cgroups.Slice(slice, dir), &specs.LinuxResources{
```

Comment:
This includes a code change as well.

Reply:
Yes, there will be an error if this place is not changed:
```go
panic: interface conversion: *cgroups.SystemdController is not cgroups.pather: missing method Path
	panic: interface conversion: *cgroups.SystemdController is not cgroups.pather: missing method Path [recovered]
	panic: interface conversion: *cgroups.SystemdController is not cgroups.pather: missing method Path
```
for more information, you can see: [github action](https://github.com/cilium/tetragon/actions/runs/4657861231/jobs/8242871899#step:6:257)
This is because the delete func of containerd/cgroups: v1.1.0 add the step: [(kernel prevents cgroups with running process from being removed, check the tree is empty)](https://github.com/cilium/tetragon/pull/904/files#diff-b257b8fefecb3b43a1f9f34094e8f54541203d70cd1c35142b437522adbe03c5R232), while [*cgroups.SystemdController is not cgroups.pather: missing method Path](https://github.com/cilium/tetragon/pull/904/files#diff-b257b8fefecb3b43a1f9f34094e8f54541203d70cd1c35142b437522adbe03c5R372). So maybe we need to change the code Systemd to V1.

Reply:
Thanks a lot for all these details :) (cc @tixxdz)